### PR TITLE
[DOC-DE] Components

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: content/de/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+### Container Runtime
+
+Die Containerlaufzeit ist die Software, die für das Ausführen von Containern verantwortlich ist.
+Kubernetes unterstützt mehrere Laufzeiten: [Docker](http://www.docker.com), [containerd](https://containerd.io), [cri-o](https://cri-o.io/), [rktlet](https://github.com/kubernetes-incubator/rktlet) und jede Implementierung des [Kubernetes CRI (Container Runtime Interface)](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/container-runtime-interface.md).
+
+## Addons
+
+Addons sind Pods und Dienste, die Clusterfunktionen implementieren. Die Pods können verwaltet werden
+durch Deployments, ReplicationControllers, und so wieter. 
+Namespace-Addon-Objekte werden im Namespace `kube-system 'erstellt.
```
Comment (rlenferink) [inline content/de/docs/concepts/overview/components.md]: Typo

```suggestion
Namespace-Addon-Objekte werden im Namespace `kube-system` erstellt.
```

**Comment by rlenferink** [comment]
Ping @bene2k1 for processing feedback :)

**Comment by bene2k1** [comment]
@rlenferink should be good now

**Comment by rlenferink** [comment]
Thanks for the changes @bene2k1

Having a look at the [components page](https://deploy-preview-13531--kubernetes-io-master-staging.netlify.com/de/docs/concepts/overview/components/) it is using the glossary keywords now, so that's good :) 

Can you translate the following as well, as it looks a bit strange showing half the page in English and the other half in German:
```
content/en/docs/reference/glossary/etcd.md
content/en/docs/reference/glossary/kube-apiserver.md
content/en/docs/reference/glossary/kube-controller-manager.md
content/en/docs/reference/glossary/kube-scheduler.md
content/en/docs/reference/glossary/kubelet.md
```

Many thanks again :)

**Comment by bene2k1** [comment]
@rlenferink The missing parts are translated and it should be good now:
https://deploy-preview-13531--kubernetes-io-master-staging.netlify.com/de/docs/concepts/overview/components/

**Pull request merged**